components/scripts/gradle/03-validate-local-build-caching-different-locations.sh

```diff
@@ -19,7 +19,6 @@ readonly SCRIPT_NAME
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")"; cd -P "$(dirname "$(readlink "${BASH_SOURCE[0]}" || echo .)")"; pwd)"
 readonly SCRIPT_DIR
 readonly LIB_DIR="${SCRIPT_DIR}/lib"
-readonly BUILD_SCAN_SUPPORT_TOOL_JAR="${SCRIPT_DIR}/build-scan-support-tool.jar"
```

Comment:
Please make sure the interactive text for this experiment also uses the following text (like for exp 2):

```
Now that the second build has finished successfully, you are ready to measure in
Gradle Enterprise how well your build leverages Gradle’s local build caching
functionality for the invoked set of Gradle tasks.
```

Reply:
Updated in 889a0f7ee8741414def3d71641c148cb07e8d2e9

See: https://github.com/gradle/gradle-enterprise-build-validation-scripts/blob/058444a1dbc5c2752a5819db275f507ca476029e/components/scripts/gradle/03-validate-local-build-caching-different-locations.sh#L309-L343